specification/archSpec/base/ditamap-attributes.dita

```diff
@@ -50,8 +50,7 @@
 						available in topics on the <xmlelement>linklist</xmlelement> and
 							<xmlelement>linkpool</xmlelement> elements, where it has the same
 						behavior. Where the <xmlatt>collection-type</xmlatt> attribute is available
-						on elements that cannot directly contain elements (such as
-							<xmlelement>reltable</xmlelement> or <xmlelement>topicref</xmlelement>),
+						on elements that cannot directly contain elements,
 						the behavior of the attribute is reserved for future use. </dd>
```

Comment:
At this point, after clearing out all of our other "reserved for future use" language, sort of feel like we should use the terminology "undefined" here -- "the behavior of the attribute is undefined." That is just as accurate as "reserved for future use", but doesn't imply that we have a plan for it down the line.